src/display/text_layer.js

```diff
@@ -104,10 +105,10 @@ var renderTextLayer = (function renderTextLayerClosure() {
       left = tx[4] + (fontAscent * Math.sin(angle));
       top = tx[5] - (fontAscent * Math.cos(angle));
     }
-    textDiv.style.left = left + 'px';
-    textDiv.style.top = top + 'px';
-    textDiv.style.fontSize = fontHeight + 'px';
-    textDiv.style.fontFamily = style.fontFamily;
+    textDivProperties.style = 'left: ' + left + 'px; top: ' + top +
+                              'px; font-size: ' + fontHeight +
+                              'px; font-family: ' + style.fontFamily + ';';
```

Comment:
Sure, I'm fine with that, as long as we keep a similar comment above it (since otherwise it may look like magic that may be removed later on).